cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -810,6 +809,30 @@ declare namespace Deno {
    * @category Network
    */
   export interface DatagramConn extends AsyncIterable<[Uint8Array, Addr]> {
+    /** Joins an IPv4 multicast group. */
+    joinMulticastV4(
+      address: string,
+      networkInterface: string,
+    ): Promise<{
+      /** Leaves the multicast group. */
+      leave: () => Promise<void>;
+      /** Sets the multicast loopback option. If enabled, multicast packets will be looped back to the local socket. */
+      setLoopback: (loopback: boolean) => Promise<void>;
```

Comment:
Is the loopback status of a socket commonly changed while it's in use? Would it make sense to actually set this up as a parameter?

Reply:
I wasn't sure about this either, but as nearly every API for multicasting I've seen has this as a method call, I felt it was the safest option. 

When I read about multicast loopback, it does seem hard to imagine that you'd be flicking it on and off all the time... but I'm hesitant to make it impossible just because of my limited imagination.

Reply:
Yeah, turning this into a parameter would make f.ex. Node compat probably impossible to achieve.

That being said... Maybe the loopback option could still be placed in some options object here or there. The most relevant place might be the Dgram creation itself, since this was for the entire socket right, and not just some membership?

We can then just leave this method here as an extra API that isn't really recommended in the general case.

Reply:
I've added an extra optional loopback parameter to `Deno.listenDatagram`, and left the methods on the membership objects.